docs/x509.rst

```diff
@@ -447,6 +447,98 @@ X.509 Extensions
 
         Returns an instance of the extension type corresponding to the OID.
 
+.. class:: KeyUsage
+
+    .. versionadded:: 0.9
+
+    The key usage extension defines the purpose of the key contained in the
+    certificate.  The usage restriction might be employed when a key that could
```

Comment:
This language is actually lifted directly from the RFC. In practice certificates with key usages that don't match their EKUs are not unusual and as a result I'm unsure if, at least in the presence of an EKU, any software cares about purposes like digitalSignature, etc.